pkg/storage/olympus/saver.go

```diff
@@ -1,8 +1,16 @@
 package olympus
 
+import (
+	"github.com/gobuffalo/buffalo"
+	"github.com/gomods/athens/pkg/storage"
+)
+
+// asserts that a ModuleStore is a storage.Saver
+var _ storage.Saver = &ModuleStore{}
+
 // Save stores a module in olympus.
 // This actually does not store anything just reports cache miss
-func (s *ModuleStore) Save(module, version string, _, _ []byte) error {
+func (s *ModuleStore) Save(_ buffalo.Context, module, version string, _, _, _ []byte) error {
 	// dummy implementation so Olympus Store can be used everywhere as Backend iface
 	return nil
 }
```

Comment:
I forgot all about this package, I don't think we use it or need it anymore. Can do it here or a follow up, either is cool with me

Reply:
Ref: https://github.com/gomods/athens/issues/180

Reply:
this is just a wrapper which reads from olympus using OlympusAPI, used in `cmd/proxy/actions/cache_miss_fetcher.go`